styles/cart.css

```diff
@@ -0,0 +1,83 @@
+body{
+    margin: 0px;
+}
+.box1{ grid-area: header; }
+.box2{ grid-area: box2;
+display: grid;
+grid-template-columns: 1fr 1fr;
+gap: 1rem; }
+.box3 { grid-area: footer; }
+
+.navigation{
+    width: 98.8%;
+    padding: 0.5rem;
+    position: static; 
+    background-color: yellowgreen;
+}
+
+.nav-bar a{
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    position: relative;
+    left: 70rem;
+}
+
+.footer{
+    width: 99.5%;
+    position: static;
+    margin-bottom: 0%;
+    background-color: yellowgreen;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-areas:
+    'header header header header header header'
+    'box2 box2 box2 box2 box2 box2'
+    'footer footer footer footer footer footer';
```

Comment:
you can write it simply as 
grid-template-areas:"header"
"box2"
"footer"
It will work in the same manner

Reply:
Sure. I will update it